### PR TITLE
Nico/mainmenus

### PR DESCRIPTION
**Pull request by Nicocchi**
Ignore the layout/design.

**Volume:** _Works_
- Master
- BGM
- SFX
- Added each bus as well. Wasn't sure if there were any in the game or not

**Graphics:** _Works_
- Fullscreen / Bordered
- VSync

**Language:** _WIP_

Should we make it a drop down or a button? I have the Checkbutton rn, and trying to figure out how to make it "toggle each other" off like radio buttons.

### CODE REVIEW

File: options_menu.gd [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+func _fullscreen(button_pressed):
+	print("fullscreen")
+	if button_pressed == true:
+		DisplayServer.window_set_mode(DisplayServer.WINDOW_MODE_FULLSCREEN)
+	else:
+		DisplayServer.window_set_mode(DisplayServer.WINDOW_MODE_WINDOWED)
+
+func _borderless(button_pressed):
+	if button_pressed == true:
+		DisplayServer.window_set_mode(DisplayServer.WINDOW_MODE_EXCLUSIVE_FULLSCREEN)
```
Comment (WebFreak001) [inline options_menu.gd]: this is fullscreen not borderless

File: options_menu.gd [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+# Hardcoded because thinking of having the options menu accessable from
+# gameplay and when clicking back it will take you there and not to title
+func _back():
+	get_tree().change_scene_to_file("res://main_menu.tscn")
+
+func _fullscreen(button_pressed):
+	print("fullscreen")
+	if button_pressed == true:
+		DisplayServer.window_set_mode(DisplayServer.WINDOW_MODE_FULLSCREEN)
```
Comment (WebFreak001) [inline options_menu.gd]: this is borderless not fullscreen

**Review by WebFreak001 — CHANGES_REQUESTED** [review]

File: options_menu.gd (hunk 1 above)
Comment (Nicocchi) [reply options_menu.gd]: borderless should be deleted

File: options_menu.gd (hunk 2 above)
Comment (Nicocchi) [reply options_menu.gd]: Then why have two fullscreen modes....

Comment (Nicocchi) [reply options_menu.gd]: The only difference is exclusive fullscreen is a single window full screen mode and non-exclusive has multi window support.
[WindowMode Docs](https://docs.godotengine.org/en/latest/classes/class_displayserver.html#enum-displayserver-windowmode)

**Review by WebFreak001 — APPROVED** [review]

**Pull request merged**